### ISSUE TITLE
places aren't getting search 

### ISSUE THREAD

**Issue by KishorKokate** [udacity/AdvancedAndroid_Shushme#31]

**Comment by KishorKokate** [comment]
and recycler view doesn't show nothing.

**Comment by Sagar0-0** [comment]
Yes, because placepicker class is deprecated and currently have no official replacement for that, either we have to use some open source library or we have to write the whole boilerplate code for that. Checkout this codelab for the same: https://developers.google.com/codelabs/maps-platform/location-places-android?index=..%2F..index#0
